cf-promises/cf-promises.c

```diff
@@ -469,6 +479,11 @@ static void ShowVariablesFormatted(EvalContext *ctx)
             var_value = "<non-printable>";
         }
 
+
+        StringSet *tagset = EvalContextVariableTags(ctx, v->ref);
+        Buffer *tagbuf = StringSetToBuffer(tagset, ',');
+
+        char *line;
```

Comment:
I think the diff for this whole file should be moved out and into a separate pull request. It's not really related to the network discovery stuff, and it would be small and quick to get in separately.